mysql-test/suite/rocksdb/t/index_merge_rocksdb2-master.opt

```diff
@@ -0,0 +1 @@
+--loose-rocksdb_strict_collation_check=off --binlog_format=row --log-bin --transaction-isolation='READ-COMMITTED'
```

Comment:
true, binlog options again are probably not needed here, but, trx iso I think is as IIRC the test includes files that use multiple connections...I'll have to look to be sure.